ote_cli/ote_cli/utils/telemetry.py

```diff
@@ -0,0 +1,92 @@
+# nosec
+
+# Copyright (C) 2021-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+"""
+Utilities for OpenVINO telemetry
+"""
+
+# pylint: disable=broad-except, ungrouped-imports
+import json
+
+from ote_cli import get_version
+
+try:
+    import openvino_telemetry as tm
+except ImportError:
+    from ote_cli.utils import telemetry_stub as tm
+
+
+__TM_CATEGORY_OTX = "otx"
+__TM_MEASUREMENT_ID = "UA-17808594-29"
```

Comment:
Is it okay to use ID on code, explicitly?

Reply:
i'm not sure but other tool like Datumaro also include the ID as a code.